### ISSUE TITLE
"py" works, but "python" doesn't

### ISSUE THREAD

**Issue by Gernatch** [rgbkrk/atom-script#1051]
It seems that "python" doesn't seem to launch python anymore from CMD (after adding the environment variable). Instead, "py" launches python.
![python-py](https://cloud.githubusercontent.com/assets/7558344/17860077/210c6900-685a-11e6-8e93-a899844e641c.png)

This is a problem, because I think **script** is calling "python." Is there a way to change this?

**Comment by nixel2007** [comment]
You can use `script options` view, create a profile or add `bat`-file with `python` call to you path.

Is `py` the new name for `python` executable? some new version?

**Comment by Gernatch** [comment]
No, the executable is still "Python.exe" so I am a bit stumped at this point. 

Seems like this is something new(er) to Python for version selection. I could be wrong on whether it is new. But you can do the following to specify a version:

py -2
py -3

It may be doing this because I have 2.7 and 3.5 installed.

**Comment by Gernatch** [comment]
@nixel2007 Not sure how to actually do what you've suggested. Where is the script options view, or profile creation? I don't really want to use a bat file preferably.

**Comment by nixel2007** [comment]
@Gernatch 
![optionsview](https://cloud.githubusercontent.com/assets/1132840/17891996/9f12b960-6947-11e6-9984-70ca51098546.gif)

**Comment by Gernatch** [comment]
Big thanks. This worked.

**Comment by gato001k1** [comment]
go to windows folder search for py.exe and rename it to python.exe thats it
